sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.controller.service.ts

```diff
@@ -165,7 +168,7 @@ export class EducationProgramOfferingControllerService {
     return {
       ...payload,
       locationId,
-      studyBreaks: payload.breaksAndWeeks?.studyBreaks,
+      studyBreaks: payload.studyBreaks,
```

Comment:
so, from now we are not saving the below values to DB?
```
{
    "totalDays": 104,
    "totalFundedWeeks": 15,
    "fundedStudyPeriodDays": 104,
    "unfundedStudyPeriodDays": 0
}
```

Reply:
The values are still saved but they did not come from the form.io submission.
Since we did the offering bulk upload these values started to be calculated by the validation process (because they are needed for some validations) and then saved to DB from the output of the calculations.
The change here is that these values and the study breaks are independent of each other, we are saving to the same DB column but they are independent. 
So now we have `studyBreaks` and `studyPeriodBreakdown`.